views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -800,14 +800,14 @@ define([
             if (this._delayCallback || this.countdown && this.countdown.isDisplayed()) {
                 return;
             }
-            _.invoke(this.controls, 'updateState');
+            _.invokeMap(this.controls, 'updateState');
```

Comment:
To fix a failing test for CI; explanation and isolated PR is here: 
https://github.com/oat-sa/extension-tao-itemqti-pci/pull/393